ant_plus_sensor/ant_plus_logger.js

```diff
@@ -167,23 +166,27 @@ async function heartRateConnect(antPlus) {
   // Announce we're online once ANT+ stick is also connected
   mqttClient.publish(statusTopic, JSON.stringify(onlineMsg), { retain: true });
 
-  mqttClient.subscribe([startTopic, stopTopic]);
-  mqttClient.on('message', (topic) => {
+  mqttClient.subscribe(v3StartTopic);
+  mqttClient.on('message', (topic, payload) => {
     winston.info(`Topic fired: ${topic}`);
-    switch (topic) {
-      case startTopic:
+
+    if (topic === v3StartTopic) {
+      const msg = JSON.parse(payload);
+      winston.info(
+        `Received data from topic ${topic}: ${JSON.stringify(msg.start)}.`,
+      );
```

Comment:
Nit: We  already have a log statement on line 171, this one would just be repetitive.

Reply:
I think I left that there from when I was testing, as it is redundant I've removed it from the code.